### ISSUE TITLE
Account Lines (Page) - Account Groups

### ISSUE THREAD

**Issue by tscottjendev** [Jersey-Electricity-plc/jwissues#52]
Does this information need to be on the page?
Can we move it to a factbox?

**Comment by RPCJendev** [comment]
Do you mean Account Group Code??

**Comment by tscottjendev** [comment]
Yes, Account Group Master Account, Account Group Code, Account Name.  They are all Flowfields from the Account record.
![image](https://github.com/Jersey-Electricity-plc/jwissues/assets/102963427/cb8af4c5-57c4-4251-bba0-3621b4756982)

**Comment by RPCJendev** [comment]
Factbox would be grand
